src/main/java/bio/terra/service/filedata/google/firestore/FireStoreUtils.java

```diff
@@ -143,6 +149,48 @@ <V> void scanCollectionObjects(
                 ApiFuture<QuerySnapshot> querySnapshot = xn.get(query);
                 return querySnapshot.get().getDocuments();
               },
+              "scanCollectionObjectsForDelete",
+              " scanning " + batchSize + " items for collection id: " + collectionId);
+      batchCount++;
+      if (!documents.isEmpty()) {
+        logger.info("Visiting batch " + batchCount + " of ~" + batchSize + " documents");
```

Comment:
There's no way to easily obtain how many total documents we're processing, right? Just have to fetch in batches until we have none left? (Or maybe the total is logged earlier?)

I ask because I find logs like this the most helpful when they clearly indicate your progress towards a goal.

Reply:
Yeah I know, it stinks but there's no way to get the full count without making another full scan query :(

Reply:
No problem, I figured that was the case!